src/clusterfuzz/_internal/bot/tasks/symbolize_task.py

```diff
@@ -55,9 +54,9 @@ def execute_task(testcase_id, job_type):
   if error:
     # TODO(metzman): Fix all this, symbolize task needs to become a utask.
     # Because this is trusted, we can trust the error.
-    all_errors = uworker_handle_errors.get_all_handled_errors()
+    assert setup.ERROR_HANDLER.is_handled(error)
```

Comment:
Why not let the valueerror happen instead of having an assert failure?

Reply:
Good point, and also this is obsolete since the original code has been deleted.